### PR TITLE
optimizing images

### PR DESCRIPTION
**Pull request by pelikhan**
Just shrinking the files.

### CODE REVIEW

**Comment by pelikhan** [comment]
@abchatra large PR.

File: pxtarget.json [hunk 1]
```diff
@@ -352,6 +352,10 @@
         "invertedMonaco": true,
         "monacoToolbox": true,
         "selectLanguage": true,
+        "availableLocales": [
+            "en",
+            "de"
```
Comment (abchatra) [inline pxtarget.json]: Do we have german translated?

Comment (ganicke) [reply pxtarget.json]: image

Comment (pelikhan) [reply pxtarget.json]: Yep. We have some very active crowdin users for German.

Comment (jwunderl) [reply pxtarget.json]: re: https://github.com/microsoft/pxt-adafruit/issues/1136

**Pull request merged**